src/cocoa/toga_cocoa/window.py

```diff
@@ -222,7 +223,18 @@ def set_title(self, title):
         self.native.title = title
 
     def set_position(self, position):
-        pass
+
+        x, y = position
+        screen = self.native.screen
+        frame = screen.frame
+
+        # From the bottom left corner of the screen to the bottom left corner
+        # of the window.
+        # self.native.setFrameOrigin(NSPoint(x, y))
+
+        # From the top left corner of the screen to the top left corner
+        # of the window.
+        self.native.setFrameTopLeftPoint(NSPoint(x, frame.size.height - y))
```

Comment:
I wasn't able to find any cases where this would *break* - did you have any specific cases in mind @samschott?

However, there's definitely some problems where y < (height of the title bar). Setting position to (0, 0) will position you at the origin (just below the menubar)... but so will (0, 10) and (0, 20). Do we need to account for `NSMenu.menuBarHeight` and/or `NSMenu.menuBarVisible?`? Or should we just accept the fact that (0, 0) is a bit of a "false" origin?

Reply:
The problem arises mostly when having two screens with different resolutions. Have a look at this sketch as an illustration:

![IMG_0654](https://user-images.githubusercontent.com/20974290/148530468-34084000-241b-405a-9c6d-d280338e1fbe.png)

Regarding the menu bar, I would just accept that (0,0) is a false origin. In particular, when the menu bar is hidden, macOS will probably correctly position the window at the top again.

Reply:
@freakboy3742, there was a good reason why your initial code ended but being a bit more complicated — it wasn’t just to confuse us ;)

Reply:
That illustration helps a lot. So we need the coordinates of this screen's origin relative to the "main" screen. Now I get it.

Reply:
Yes, exactly :)